wolox-books/src/app/screens/register/components/register-form/register-form.component.scss

```diff
@@ -0,0 +1,55 @@
+@import '../../../../../scss/colors.scss';
+
+.register-form {
+  background-color: $porcelain;
+  border-top: 3px solid $cerulean;
+  padding: 50px 20px 20px 20px;
+  text-align: center;
+  width: 325px;
+}
+
+.register-title {
+  padding-bottom: 5px;
+}
+
+.form{
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.form-label {
+  margin-top: 10px;
+}
+
+.form-input {
+  border-radius: 6px;
+  height: 30px;
+  margin:5px 0;
+  width: 100%;
+}
+
+.line{
+  border-top: 1px solid $silver;
+}
+
+%buttons-shared {
```

Comment:
I think it would be a good idea to abstract the basic styles that could be used in another part of the application. This includes the label and input styles and the buttons styles.